.vsts-dotnet.yml

```diff
@@ -77,6 +77,13 @@ stages:
       inputs:
         dropName: $(VisualStudio.DropName)
 
+    - task: PowerShell@2
+      inputs: # This is a temporary fix to get msbuild onboarded with v3 publishing. This will be resolved soon ->https://github.com/dotnet/arcade/issues/6827
+        targetType: 'inline'
+        script: |
+          Write-Host "Overwriting BUILD_REPOSITORY_URI."
+          Write-Host "##vso[task.setvariable variable=BUILD_REPOSITORY_URI;]https://dnceng@dev.azure.com/dnceng/internal/_git/dotnet-msbuild"
```

Comment:
Was BUILD_REPOSITORY_URI being used before?

Reply:
It is used when generating the asset manifest and during publishing build assets...